gef-ui/templates/partials/facilities.njk

```diff
@@ -0,0 +1,94 @@
+{% extends "index.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block pageTitle %}Facilities{% endblock %}
+
+{% block content %}
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      }
+    }) }}
+  {% endif %}
+
+  {% if status !== 'change' %}
+    {{ govukBackLink({
+        text: "Back to previous page",
+        href: "/gef/application-details/" + applicationId,
+        attributes: {
+          'data-cy': 'back-link'
+        }
+      }) }}
+    {% endif %}
+
+  <header data-cy="application-details-page">
+    <span class="govuk-caption-xl" data-cy="heading-caption">
+      Add a facility
+    </span>
+  </header>
```

Comment:
I don't know the design, but it looks like this should be a heading tag?

Reply:
I have been following https://design-system.service.gov.uk/styles/colour/ You can see that when they use a heading Caption they use a span tag, while the main title is in a heading tag. Let me know your thoughts

Reply:
I'm not following, if it's top level heading it should be h1/h2 tag https://design-system.service.gov.uk/styles/typography/

easy to change, happy to merge either way